ibis/config.py

```diff
@@ -565,29 +555,35 @@ def _warn_if_deprecated(key):
 
 
 def _build_option_description(k):
-    """ Builds a formatted description of a registered option and prints it """
-
+    """Builds a formatted description of a registered option and prints it."""
     o = _get_registered_option(k)
     d = _get_deprecated_option(k)
 
-    s = u('%s ') % k
+    buf = ['{} '.format(k)]
```

Comment:
Perhaps using `StringIO` instead of joining a list of strings?

Reply:
I find the list of strings approach more readable. Any particular reason to use StringIO?